stream-chat-android/src/main/java/com/getstream/sdk/chat/ImageLoader.kt

```diff
@@ -19,8 +19,8 @@ import coil.transform.RoundedCornersTransformation
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.withContext
 
-internal object ImageLoader {
-    suspend fun getBitmap(
+public object ImageLoader {
+    public suspend fun getBitmap(
```

Comment:
Looks like this could be `@InternalStreamChatApi` so that we don't expose it completely (which we then have to opt into in our new module in the `build.gradle` config, like with some existing ones)

Reply:
Done